practice-one/src/styles/sections/introduction.css

```diff
@@ -105,3 +105,54 @@
   vertical-align: middle;
   cursor: pointer;
 }
+
+/* Style for desktop screen under 1300px*/
+@media (max-width: 1300px) {
```

Comment:
Why 1300px?

Reply:
at screen width under 1300px layout will be broken, so i choose that breakpoint